File: StarRentalWebsite/src/app/models/payment-info.ts
export interface PaymentInfo {
  payment_id: number;
  cardholder_name: string;
  card_number: string;
  exp_month: number;
  exp_year: number;
  cvv: string;
  billing_address: string;
  city: string;
  state: string;
  zip_code: string;
  country: string;
  user_id: number;
}

File: StarRents/StarRentalMng/schemas/payment_info.py
from pydantic import BaseModel
from typing import Optional


class PaymentInfo(BaseModel):
    payment_id: int
    cardholder_name: str
    card_number: str
    exp_month: str
    exp_year: str
    cvv: str
    billing_address: Optional[str] = None
    city: Optional[str] = None
    state: Optional[str] = None
    zip_code: Optional[str] = None
    country: Optional[str] = None
    user_id: Optional[int] = None

class PaymentID(BaseModel):
    payment_id: int